plugins/xtrace/wiring.go

```diff
@@ -3,7 +3,7 @@
 //  1. A modifier plugin to wrap the service with an XTrace wrapper to generate XTrace compatible traces/logs by correctly propagating baggage across service boundaries. Automatically generates and include an xtrace-server instance in a Blueprint application.
 //  2. A plugin to define an xtrace-based logger for a process. Log events are added as reports to the currently active xtrace task, if one exists. If no such task exists, then no events are logged.
 //
-// Once the application is instrumented with these plugins, traces will be generated and collected by the xtrace-server. Traces will consist of xtrace events generated by plugins 2 and 3.
+// Once the application is instrumented with these plugins, traces will be generated and collected by the xtrace-server. Traces will consist of xtrace events generated by plugins 1 and 2.
```

Comment:
I don't like the "plugins 1 and 2" phraseology.  I think you can just drop the second sentence.